pkg/informer/sample-allowdeny.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I was only using the json to get the string format for Allowed/DeniedResources. Would you prefer to have yaml fiels for each test case instead of configmap objects?

Reply:
no, the configmap objects are perfect.

Reply:
okay, we can delete this json file all together.